src/Compilers/VisualBasic/Portable/Symbols/MissingMetadataTypeSymbol.vb

```diff
@@ -167,23 +172,50 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 End Get
             End Property
 
-            Public Overrides ReadOnly Property SpecialType As SpecialType
+            Private ReadOnly Property TypeId As Integer
```

Comment:
I was referring to TypeId, you are correct, it is new.

---
In reply to: [77396335](https://github.com/dotnet/roslyn/pull/13542#discussion_r77396335) [](ancestors = 77396335)